subsys/spm/Kconfig

```diff
@@ -9,8 +9,10 @@ menu "SPM"
 config SPM
 	bool "Use Secure Partition Manager"
 	default y if TRUSTED_EXECUTION_NONSECURE
+	imply ARM_FIRMWARE_USES_SECURE_ENTRY_FUNCS
```

Comment:
select, perhaps? Isn't it better? It will give a warning if anyone tries to build with SPM but selects the option to "n" in their prj.conf....

Reply:
If they explicitly set it to "n" it probably means they know they won't need it. I think that's a valid use case.